core/src/Pos/Core/Update/VoteId.hs

```diff
@@ -0,0 +1,26 @@
+{-# OPTIONS_GHC -fno-warn-orphans #-}
```

Comment:
Is that still needed?

Reply:
Yes, unless we change `VoteId` to a `newtype`!